packages/api/src/utils/operations.ts

```diff
@@ -1,29 +1,10 @@
 import { TRPCError } from '@trpc/server';
 
-export function addDefaultValues<T, F>(
-	input: T[],
-	getDefaultValue: (input: T) => F,
-) {
-	return input.map((v) => getDefaultValue(v));
-}
-
 export async function findOrThrowNotFound<T>(
-	find: () => Promise<T | null>,
+	find: () => Promise<T | null | undefined>,
```

Comment:
We should look to moving to only null or only undefined for the code base in the future